src/OnboardingSPA/components/LivePreview/SelectableCard/stylesheet.scss

```diff
@@ -109,3 +110,14 @@ $main-border-main: var(--nfd-onboarding-primary-alt);
 		}
 	}
 }
+
+.live-preview {
+	&-selected-check {
+		z-index: 2 !important;
+	}
+
+	&-container {
+		overflow: hidden;
+	}
+}
+
```

Comment:
Again selectable card css should be inside `.live-preview--selectable-card` we don't want to globally modify live preview here

Reply:
Actually this was intentional cause what I saw was the same CSS for Selected Card and Live Preview is duplicated in Multiple Places and due to which while changing one value all the places had to be modified, instead I made this CSS which would override them and then even if someone forgets to add this it will automatically be added.

Reply:
Removed It.